forge-gui/res/cardsfolder/upcoming/ambitious_dragonborn.txt

```diff
@@ -0,0 +1,8 @@
+Name:Ambitious Dragonborn
+ManaCost:3 G
+Types:Creature Dragon Barbarian
+PT:0/0
+K:etbCounter:P1P1:X:no Condition:CARDNAME enters the battlefield with X +1/+1 counters on it, where X is the greatest power among creatures you control and creature cards in your graveyard.
+SVar:X:Count$ValidGraveyard,Battlefield Creature.YouCtrl$GreatestPower
```

Comment:
@tool4ever this syntax doesn't look right to me... but I'm not 100% sure

Reply:
TRT and I had a discussion about this. I was reading the card wrong the whole time, but eventually it worked as it's supposed to.